src/services/art/find.js

```diff
@@ -9,36 +9,35 @@ export const getArts = async ({
   page = 1,
   pageSize,
   searchTerm,
+  username,
   sort,
   locale,
 }) => {
-  let filters = {}
-
-  const searchFilter = {
-    $or: [
-      {
-        artist: {
-          user: {
-            username: {
-              $containsi: searchTerm,
-            },
-          },
-        },
-      },
-      {
-        title: {
-          $containsi: searchTerm,
+  const userFilter = {
+    artist: {
+      user: {
+        username: {
+          $containsi: searchTerm || username,
         },
       },
-    ],
+    },
   }
 
-  const categoryObj = qs.parse(categories)
-
-  if (categoryObj?.[0]) {
-    filters = { ...(searchFilter || {}), categories: { code: { $in: Object.values(categoryObj) } } }
+  const titleFilter = {
+    title: {
+      $containsi: searchTerm,
+    },
   }
 
+  const searchFilter = username
+    ? userFilter
+    : searchFilter && {
+        $or: [userFilter, titleFilter],
+      }
```

Comment:
Should it be `searchTerm`? I see errors while building and I think it's because of this line. Could you test the searching art manually after changing this line?

```suggestion
  const searchFilter = username
    ? userFilter
    : searchTerm && {
        $or: [userFilter, titleFilter],
      }
```

<img width="1008" alt="Screenshot 2022-05-13 at 11 45 40" src="https://user-images.githubusercontent.com/22167684/168257776-0438d8a4-2c27-424e-bf2d-0f026793a2b1.png">